next.config.mjs

```diff
@@ -0,0 +1,32 @@
+// /** @type {import('next').NextConfig} */
+// const nextConfig = {
+//   reactStrictMode: true,
+//   swcMinify: true,
+// };
+
+// module.exports = nextConfig;
+
+import remarkFrontmatter from "remark-frontmatter";
+
```

Comment:
`mjs` nai use garna parne any specific reason?

Reply:
It doesn't work without the `mjs` extension probably due to the `import` statement. Can't convert it to `require` either.